dask/dataframe/io.py

```diff
@@ -462,14 +462,18 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
                  "must preserve the order of its input")
 
     # handle lock default based on wether we're writing to a single entity
+    _actual_get = get if get else dask.get
```

Comment:
Dask.get is just an alias to `dask.async.get_sync`, the single threaded scheduler.  We've chosen this historically to avoid multi-threaded use of this function when it was highly sequential.  Otherwise the dataframes-to-be-written end up piling up waiting to be written to disk, causing some memory blowup.  If we're no longer in this situation then it might make sense to use threaded.get or multiprocessing.get or whatever the user provides.